### ISSUE TITLE
failed to load ocrorot.SkewEstimator

### ISSUE THREAD

**Issue by bliu3650** [tmbdev-tutorials/das2018-tutorial#5]
All others models along the full OCR pipeline can be loaded as below way except the model for SKEW.
Thanks.

My Environment: 
PyTorch v0.3.1 without CUDA installed on Ubuntu 16.04.

Error:
Python 2.7.15 |Anaconda, Inc.| (default, May  1 2018, 23:32:55)
[GCC 7.2.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import scipy.ndimage as ndi
>>> import ocrorot
>>> sk = ocrorot.SkewEstimator("logskew-000015808-000132.pt")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "ocrorot/rotation.py", line 80, in __init__
    RotationEstimator.__init__(self, mname)
  File "ocrorot/rotation.py", line 40, in __init__
    self.model = torch.load(mname, map_location='cpu')
  File "/home/bliu29/anaconda2/lib/python2.7/site-packages/torch/serialization.py", line 267, in load
    return _load(f, map_location, pickle_module)
  File "/home/bliu29/anaconda2/lib/python2.7/site-packages/torch/serialization.py", line 420, in _load
    result = unpickler.load()
ImportError: No module named layers

**Comment by ghost** [comment]
@BrianLiu3650 I got the same issue, but i solve it !

1)  make layers.py to your ocrorot library(same dir roatiation.py)

```
# Copyright (c) 2017 NVIDIA CORPORATION. All rights reserved.
# See the LICENSE file for licensing terms (BSD-style).

import torch
import pytorch_fft.fft.autograd as afft
from torch import nn
from torch.autograd import Variable


class Spectrum(nn.Module):
    def __init__(self, nonlin="logplus1"):
        nn.Module.__init__(self)
        self.fft2d = afft.Fft2d()
        self.nonlin = nonlin

    def forward(self, x):
        y = Variable(torch.zeros(x.size()).cuda())
        #print type(x), type(y)
        #print type(x.data), type(y.data)
        re, im = self.fft2d(x, y)
        #print type(re), type(im)
        #print type(re.data), type(im.data)
        mag = (re**2 + im**2)
        if self.nonlin == None:
            return mag
        elif self.nonlin == "logplus1":
            return torch.log(1.0 + mag)
        elif self.nonlin == "sqrt":
            return torch.mag ** .5
        else:
            raise Exception("{}: unknown nonlinearity".format(self.nonlin))

    def __repr__(self):
        return "Spectrum"
```


2) pip install pytorch-fft

I hope it works

**Comment by bliu3650** [comment]
@a41888936 It really works! Thanks!
I missed the readme of ocrorot about the use of "Spectrum". So far, I still do not know much about it.
After I got your tips, I passed the full pipeline flow, and then checked around and found that this layers was imported in the ocroskew-train. I am thinking this layers.py might be ignored by the author @tmbdev or something else. Thanks again for helping me out from here.
